src/helix-ui/styles/mixins.less

```diff
@@ -5,7 +5,7 @@
   opacity: 1;
 }
 
-#Util {
+#Mix {
```

Comment:
* file renamed to `mixins.less`
* namespace renamed from `#Util` to `#Mix` for use in including files